osu.Game/Beatmaps/BeatmapManager.cs

```diff
@@ -265,9 +265,14 @@ public virtual void Save(BeatmapInfo info, IBeatmap beatmapContent, ISkin beatma
 
                     // grab the original file (or create a new one if not found).
                     var fileInfo = setInfo.Files.SingleOrDefault(f => string.Equals(f.Filename, beatmapInfo.Path, StringComparison.OrdinalIgnoreCase)) ?? new BeatmapSetFileInfo();
-
+                    
                     // metadata may have changed; update the path with the standard format.
                     beatmapInfo.Path = $"{metadata.Artist} - {metadata.Title} ({metadata.Author}) [{beatmapInfo.Version}].osu";
+
+                    // removing incompatible characters before exporting
```

Comment:
I'd write something like:

```suggestion
                    // remove incompatible characters from path
```

If you look at the other comments, they all use the present tense.